include/seastar/core/file.hh

```diff
@@ -61,6 +61,14 @@ struct directory_entry {
     std::optional<directory_entry_type> type;
 };
 
```

Comment:
> Add APIs to get group details and to change ownership of file.

Please enrish the patch changelog with the details.

Reply:
> > Add APIs to get group details and to change ownership of file.
> 
> Please enrich the patch changelog with the details.

Thanks, I have added details to the commit log